.travis.yml

```diff
@@ -2,7 +2,7 @@ language: nix
 
 before_deploy:
   - mkdir pages
-  - cp -rf result/* pages/
+  - cp -f $(readlink ./result) pages/main.pdf
```

Comment:
Is readlink necessary? Doesn't `cp` follow symlinks?

Reply:
Yes, you're right: it'd work without `readlink`. I'd still suggest to leave it there for clarity and because it shouldn't make a difference, right?

Reply:
I think that it's a symlink is just an implementation detail of the build. We don't need to explicitly say that here.
Also is the `-f` necessary? I don't think so, but I would be okay with leaving that, just for good measure.

Reply:
I don't think that `git` stores anything other then the excute-permission bit, but if the target file were marked as read only the `-f` would be necessary. I'd rather not rely on such things and make my goals explicit.